apps/wfo-ui/pages/service-tickets/[serviceTicketId].tsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
```

Comment:
This folder can be moved to surf specific app after rebasing

Reply:
> Rene:
> @ricardovdheijden / @DutchBen : I don't know if it's a good idea to now have a new "wfo-surf-ui" top level app. I saw it's already merged; but this leads to a lot of duplication and extra maintenance: you now have to sync both folders. At first, till march, this will be only one extra route: "/tickets" which we could also just hide based on an env var.
> 
> Personally I would've tried to avoid having 2 duplicate apps as long as possible...
> 
> So something to keep in mind when reviewing code. Now commits in "wfo-ui" should be duplicated in "wfo-surf-ui"!

Should I wait moving the service-tickets folder to wfo-ui or don't move at all, until a decision is made on the situation above ?

Reply:
For now we stick to the wfo-ui-surf app next to the normal app to avoid too much SURF specific code in the demo app. You can rebase from main from now on